dawn/src/dawn/Serialization/IIRSerializer.cpp

```diff
@@ -74,8 +75,6 @@ static void setCache(proto::iir::Cache* protoCache, const iir::Cache& cache) {
   case iir::Cache::IOPolicy::unknown:
     protoCache->set_policy(proto::iir::Cache_CachePolicy_CP_Unknown);
     break;
-  default:
-    dawn_unreachable("unknown cache policy");
```

Comment:
why are some of them getting removed while others are replaced?

Reply:
Because there's no default option needed here. All cases are accounted for in the switch. The ones that I kept the default on are almost all proto:: enumerations that have a hidden `invalid` option too (at least clang told me so).

Reply:
having this default future proofs this though (adding a new option and forgetting it here would prompt an error instead of silently ignoring it, right?

Reply:
Decided to leave this as is, so there will be a compiler warning/error if new enum values are added and new cases are not added here.